### DISCUSSION TITLE
How to insert default datetime in Browse Data?

### DISCUSSION THREAD

**Discussion by navadev** [sqlitebrowser/sqlitebrowser#3726] [Q&A] [answered]
Is there a button to add a default datetime (i.e. current datetime) instead of NULL when manually editing the data in a table?

**Comment by mgrojo** [comment, marked as answer]
There's no button, but there is an easy way. Change the editor mode of the "Edit Database Cell" dock to "Evaluation" and write `CURRENT_TIMESTAMP` (the editor completes) then save it pressing the Apply button or Ctrl+Return.

If you want to do the same for several cells, deactivate the automatic mode of the editor to keep the evaluation mode for all the cells.
